Neos.Flow/Tests/Unit/Http/CookieTest.php

```diff
@@ -255,6 +255,60 @@ public function isHttpOnlyReturnsHttpOnlyFlag()
         $this->assertFalse($cookie->isHttpOnly());
     }
 
+    /**
+     * @test
+     */
+    public function SameSiteReturnsNull()
+    {
+        $cookie = new Cookie('foo', 'bar');
+        $this->assertNull($cookie->getSameSite());
+    }
+
+    /**
+     * @test
+     */
+    public function SameSiteReturnsNone()
+    {
+        $cookie = new Cookie('foo', 'bar', 0, null, 'neos.io', '/', true, false, Cookie::SAMESITE_NONE);
+        $this->assertSame(Cookie::SAMESITE_NONE, $cookie->getSameSite());
+    }
+
+    /**
+     * @test
+     */
+    public function SameSiteNoneEnablesSecure()
+    {
+        $cookie = new Cookie('foo', 'bar', 0, null, 'neos.io', '/', false, false, Cookie::SAMESITE_NONE);
+        $this->assertTrue($cookie->isSecure());
```

Comment:
@mficzel here is the test for the case you haven't test.

Reply:
I know and i am confident that it works. Just wanted to state that i could not verify 100% on my setuo